src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,52 @@
 package mate.jdbc;
 
+import java.util.Optional;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Driver driver = new Driver();
+    private static final Long INDEX_1 = 1L;
+    private static final Long INDEX_2 = 2L;
+    private static final String LICENCE_NUMBER = "21214";
+    private static final String DRIVER_NAME = "Anatoliy";
+    private static Injector injector =
+            Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        deleteDriverById(INDEX_1, driverService);
+        updateDriver(INDEX_1, LICENCE_NUMBER, DRIVER_NAME, driverService);
+        getDriverById(INDEX_1, driverService);
+        createDriver(INDEX_2, LICENCE_NUMBER, DRIVER_NAME, driverService);
+    }
+
+    private static void deleteDriverById(Long id,
+                                         DriverService driverService) {
+        driverService.delete(id);
+    }
+
+    private static Optional<Driver> getDriverById(Long id,
+                                                  DriverService driverService) {
+        return Optional.ofNullable(driverService.get(id));
+    }
```

Comment:
DriverService already thrown exception if there isn't such id. Why we should cover result with Optional again?